docker/postlaunchconfig_update.sh

```diff
@@ -89,8 +89,8 @@ if [ "${LOG_LEVEL}" == "DEBUG" ]; then
     DEFAULT debug True
     openstack-config --set /etc/keystone/keystone.conf \
     DEFAULT insecure_debug True
-    openstack-config --set /etc/keystone/keystone.conf \
-    wsgi debug_middleware True
+    # openstack-config --set /etc/keystone/keystone.conf \
+    # wsgi debug_middleware True
```

Comment:
Leftover comments?

Reply:
Fixed in https://github.com/telefonicaid/fiware-keystone-spassword/pull/237/commits/0524907c39d60b11754f20f2674bc346cf9ee283